docs/content/docs/connectors/table/kinesis.md

```diff
@@ -57,16 +87,21 @@ CREATE TABLE KinesisTable (
 PARTITIONED BY (user_id, item_id)
 WITH (
   'connector' = 'kinesis',
-  'stream' = 'user_behavior',
+  'stream.arn' = 'arn:aws:kinesis:us-east-1:012345678901:stream/my-stream-name',
   'aws.region' = 'us-east-2',
```

Comment:
Region in `stream.arn` and `aws.region` should match

Reply:
done